gneiss/_model.py

```diff
@@ -7,29 +7,28 @@
 # ----------------------------------------------------------------------------
 import pickle
 import abc
+import numpy as np
 
 
 class Model(metaclass=abc.ABCMeta):
 
-    def __init__(self, submodels, balances):
+    def __init__(self, Y, Xs):
         """
         Abstract container for balance models.
 
         Parameters
         ----------
-        submodels : list of statsmodels objects
-            List of statsmodels result objects.
-        balances : pd.DataFrame
-            A table of balances where samples are rows and
-            balances are columns. These balances were calculated
-            using `tree`.
+        Y : pd.DataFrame
+            Response matrix.  This is the matrix being predicted.
+            Also known as the dependent variable in univariate analysis.
+        Xs : iterable of pd.DataFrame
+            Design matrices.  Also known as the independent variable
+            in univariate analysis. Note that this allows for multiple
+            design matrices to be inputted to enable multiple data block
+            analysis.
```

Comment:
In case you are wondering why refactor this section.
We want to ultimately be able to enable multiblock regression / classification #188
So it would be great if we could pass in a single response matrix, and multiple design matrices to try to predict the regression matrix.